src/main/kotlin/org/rust/cargo/commands/Cargo.kt

```diff
@@ -53,6 +55,7 @@ class Cargo(
             .withParameters(command)
             .withParameters(additionalArguments)
             .withEnvironment(environmentVariables)
+            .withEnvironment(CargoConstants.RUSTC_ENV_VAR, pathToRustExecutable)
```

Comment:
Doesn't really matter, `withEnvironment(Map)` does a `putAll()` and `withEnvironment(String, String)` does a put.